component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -175,6 +179,78 @@ public static String getTOTPIssuerDisplayName(String tenantDomain, Authenticatio
         return issuer;
     }
 
+    private static String getIssuerFromParameters() {
+
+        if (Boolean.parseBoolean(getTOTPParameters().get(TOTPAuthenticatorConstants.TOTP_COMMON_ISSUER))) {
+            return getTOTPParameters().get(TOTPAuthenticatorConstants.TOTP_ISSUER);
+        }
+        return null;
+    }
+
+    private static String getIssuerFromContextOrRegistry(String tenantDomain, AuthenticationContext context)
+            throws TOTPException {
+
+        if (context == null) {
+            return getIssuerFromRegistry(tenantDomain);
+        } else if (context.getProperty(TOTPAuthenticatorConstants.TOTP_ISSUER) != null) {
+            return (String) context.getProperty(TOTPAuthenticatorConstants.TOTP_ISSUER);
+        }
+        return null;
+    }
+
+    private static String getIssuerFromBranding(String tenantDomain) {
+
+        BrandingPreferenceManager brandingPreferenceManager = TOTPDataHolder.getInstance()
+                .getBrandingPreferenceManager();
+        if (brandingPreferenceManager == null) {
+            return null;
+        }
+
+        try {
+            BrandingPreference responseDTO = brandingPreferenceManager.resolveBrandingPreference(
+                    BrandingPreferenceMgtConstants.ORGANIZATION_TYPE, tenantDomain,
+                    BrandingPreferenceMgtConstants.DEFAULT_LOCALE, false);
+            JsonNode brandingPreferences = new ObjectMapper().valueToTree(responseDTO.getPreference());
+
+            if (!brandingPreferences.at("/configs/isBrandingEnabled").asBoolean()) {
+                return null;
+            }
+            return brandingPreferences.at("/organizationDetails/displayName").asText();
+        } catch (BrandingPreferenceMgtException e) {
+            handleBrandingPreferenceException(e, tenantDomain);
+            if (BrandingPreferenceMgtConstants.ErrorMessages.ERROR_CODE_BRANDING_PREFERENCE_NOT_CONFIGURED.getCode()
+                    .equals(e.getErrorCode())) {
+                if (log.isDebugEnabled()) {
+                    String message = "No branding preferences exist for organization " + tenantDomain ;
+                    log.debug(message, e);
+                }
+            } else {
+                if (log.isDebugEnabled()) {
+                    String message = "Error occurred while retrieving branding preferences for organization "
+                            + tenantDomain ;
+                    log.debug(message, e);
+                }
+            }
```

Comment:
```suggestion
            if ( log.isDebugEnabled()) {
                String message = BrandingPreferenceMgtConstants.ErrorMessages.ERROR_CODE_BRANDING_PREFERENCE_NOT_CONFIGURED.getCode().equals(e.getErrorCode()) ? "No branding preferences exist for organization " + tenantDomain  : "Error occurred while retrieving branding preferences for organization "
                            + tenantDomain ;
                    log.debug(message, e);
            }
```

Reply:
I have forgotten to remove the refactored code into the method from the main method. Thanks for pointing. This was fixed with de968bc4321b6a8038eb9164341281d729a5adb1.